src/main/java/org/akip/domain/ProcessInstanceNotification.java

```diff
@@ -0,0 +1,179 @@
+package org.akip.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.akip.domain.enumeration.ProcessInstanceEventType;
+import org.akip.domain.enumeration.ProcessInstanceNotificationStatus;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDate;
+
+/**
+ * A ProcessInstanceNotification.
+ */
+@Entity
+@Table(name = "process_instance_notification")
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+public class ProcessInstanceNotification implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
+    @SequenceGenerator(name = "sequenceGenerator")
+    private Long id;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "date")
+    private LocalDate date;
```

Comment:
`date` is a type. Change for a meaningful name such as `createDate`.
Include a `readDate` as well.